tests/test_workload.py

```diff
@@ -2,12 +2,21 @@
 from uuid import uuid4
 
 import pytest
+from pydantic import ValidationError
 
 from pasqal_cloud import SDK, Workload
 from tests.test_doubles.authentication import FakeAuth0AuthenticationSuccess
 
 
 class TestWorkload:
+    @pytest.fixture
+    def workload_with_link_id(self):
+        return "00000000-0000-0000-0000-000000000002"
```

Comment:
Using 'str(UUID(int=0x2))' might be better as we prefer to use this instead of plain string to avoid missing a 0. :D